engine/src/main/java/org/terasology/network/BroadCastServer.java

```diff
@@ -0,0 +1,133 @@
+/*
+ * Copyright 2020 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.terasology.network;
+
+import java.net.DatagramPacket;
+import java.net.DatagramSocket;
+import java.net.InetAddress;
+import java.nio.charset.StandardCharsets;
+import java.util.concurrent.Executors;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class BroadCastServer {
+
+    private static boolean turnOnBroadcast;
+    private static final Logger logger = LoggerFactory.getLogger(BroadCastServer.class);
+    private  final String discoveryRequest = "DISCOVERY_REQUEST";
+    private  final String discoveryResponse = "DISCOVERY_RESPONSE";
+    private  DatagramSocket receiveSocket;
+    private  DatagramSocket sendSocket;
+
+
+    public boolean isBroadCastTurnedOn() {
+        return turnOnBroadcast;
```

Comment:
maybe use _active_ as word here? `BroadCastServer.isActive()` may be a bit more intuitive.